### ISSUE TITLE
Theme fails to deploy to Docker Container

### ISSUE THREAD

**Issue by mwilliams2014** [liferay/liferay-js-themes-toolkit#440]
At first I was unable to deploy a Theme to the Docker container at all, even manually. I had to set up a Bind Mount, and only then did it work. `npm run deploy` fails with this error:

**Comment by mwilliams2014** [comment]
I tried deploying with the deploy folder set to `1468d2530957:/opt/liferay/deploy` but it I got the same not found error. I am on Windows. Not sure if that has something to do with it.

**Comment by wincent** [comment]
It might be another one of those Windows path-handling issues. I think the first step will be to try and repro this on Linux or macOS; if it works there but not on Windows then that'll be evidence in favor of the hypothesis. We could also try just visually inspecting code that involves paths and see if anything stands out.

**Comment by mwilliams2014** [comment]
Is there anyone on the team on Mac that can test?

**Comment by wincent** [comment]
Yes, plenty of Macs around here. How urgent is this and is the problem blocking you? (Just to help us prioritize it.) cc @esanzp

**Comment by mwilliams2014** [comment]
Hey @wincent , I'm able to manually copy the Theme to my bind mounted deploy folder for testing, but we'll need to resolve this for users and docs eventually.

**Comment by mwilliams2014** [comment]
@wincent I should also note that when copying the war to the Docker container, it gives this error:

**Comment by mwilliams2014** [comment]
I can also successfully copy the Theme's WAR directly to the Docker Container's `/opt/liferay/osgi/war` folder

**Comment by crodriguezy** [comment]
Hello team, 

Is there any update concerning this issue? It seems `docker:deploy` is still failing on Windows machines. There is a Spanish customer reproducing the same when executing the gulp deploy command using 'Docker Container' as strategy:

![error-gulp-deploy](https://user-images.githubusercontent.com/29747802/98656899-853ac600-2341-11eb-9c28-378388384ac3.PNG)

They also pointed that the workaround indicated here is not working the way they expect: after manually copying the war, the `gulp watch` command does not reload the changes.

Could you please advice?

Thank you!

**Comment by wincent** [comment]
> Is there any update concerning this issue?

I don't have any new information on this one, although I believe we had an internal Slack thread on it which is visible to you @crodriguezy. In any case, I am going to transfer this over to [the new `liferay-frontend-projects` monorepo](https://github.com/liferay/liferay-frontend-projects) because we're archiving the old repo soon in order to centralize development.
